iceoryx2-bb/posix/tests/file_descriptor_set_tests.rs

```diff
@@ -155,6 +155,36 @@ fn file_descriptor_set_timed_wait_works() {
     assert_that!(result[0], eq unsafe{sut_receiver.file_descriptor().native_handle()});
 }
 
+#[test]
+fn file_descriptor_set_blocking_wait_works() {
```

Comment:
The name is a bit ambiguous. I guess you mean that a call to `blocking_wait` immediately returns with data?